spec/lib/metasploit/framework/hashes/identify_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
-require 'metasploit/framework/hashes/identify'
+require 'metasploit/framework/hashes'
 require 'bcrypt'
+include Metasploit::Framework::Hashes
```

Comment:
After making this change, I'm getting some `NoMethodError`s on verification. I'm guessing before it was working because of the wholesale include at the top. Should I go back to using the include or is there a way to adjust the spec differently?

`NoMethodError:
        undefined method `identify_hash' for Metasploit::Framework::Hashes:Module
      # ./spec/lib/metasploit/framework/hashes/identify_spec.rb:334:in `block (3 levels) in <top (required)>`